### PR TITLE
chore(deps): minor updates

### PR DESCRIPTION
**Pull request by escapedcat**
Updated minor deps.  

Skipped this one:

```sh
? Pick the packages you want to upgrade.          Current          Range            Latest
 > @heroicons/react ---------------------------- ◉ ^1.0.6 -------                  ◯ ^2.0.8 -------
 ```
 
Please see how to upgrade from v1 to v2 here: https://github.com/tailwindlabs/heroicons/releases/tag/v2.0.0

### CODE REVIEW

**Comment by escapedcat** [comment]
Can haz https://depfu.com/ pretty plz?

File: package.json [hunk 1]
```diff
@@ -54,15 +54,15 @@
     "@bitcoin-design/bitcoin-icons-react": "^0.1.9",
     "@heroicons/react": "^1.0.6",
     "axios": "^0.27.2",
-    "event-source-polyfill": "1.0.25",
+    "event-source-polyfill": "1.0.31",
```
Comment (cstenglein) [inline package.json:57]: don't want to update that; see https://github.com/fusion44/blitz_api/issues/144

**Review by cstenglein — CHANGES_REQUESTED** [review]

**Comment by cstenglein** [comment]
> Can haz https://depfu.com/ pretty plz?

see #433 

Not sure if it works perfectly, but it's a start

File: package.json (hunk 1 above)
Comment (escapedcat) [reply package.json]: Forgot about that

**Review by cstenglein — APPROVED** [review]
Thank you!

**Pull request merged**